### DISCUSSION TITLE
I can edit random project's wiki pages

### DISCUSSION THREAD

**Discussion by MatthiasQuintern** [community/community#68562] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

It seems like I can currently edit/delete random project's wiki pages.

In the vaultwarden repo, I could edit a footer, the change is visible from another device where I am not logged in.
Below https://github.com/dani-garcia/vaultwarden/wiki#get-in-touch
![image](https://github.com/community/community/assets/63054875/e47daa41-bd84-430b-a5a0-a68442ffd238)

The edit buttons are also there when not logged in, but clicking leads to 404 page.
Also works on other projects wikis.

Are these changes visible/possible for anyone else?

**Comment by radar-blip** [comment]
Yes

**Comment by MatthiasQuintern** [comment]
Thanks, I submitted a report at hackerone

**Reply by MatthiasQuintern** [reply]
Okay, so this is apparently a feature and coincidentally the three repos I tested had it enabled (I can not imagine this is a good idea, but okay).

**Comment by DNin01** [comment, marked as answer]
Note that there is a way to [enable public editing](https://docs.github.com/en/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis) on Wikis, so I don't think this is a bug.

**Reply by Cheezik** [reply]
Is there any way i can edit the Wiki without that feature being enabled? something like forking it and then pr it back?
